src/sources/dataModel/taxonomic-service.json

```diff
@@ -125,26 +261,78 @@
                 "This service is rejected, because..."
             ]
         },
+        "additionalProperty": {
+            "type": "array",
+            "description": "Generic property for additional properties not covered by schema.org",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "@type": {
+                        "type": "string",
+                        "description": "The type of the additional property",
```

Comment:
Maybe add ", which will always be `schema:PropertyValue`

Reply:
I added the schema prefix for all the PropertyValue attributes, if that is what you meant